examples/example_struct/make_bindings_via_cmake.py

```diff
@@ -14,25 +14,30 @@
 binder_source = f'{os.getcwd()}/../../source'
 pybind_source = f'{os.getcwd()}/../../build/pybind11/include'
 use_pybind_stl = True
-#use_pybind_stl = False
+# use_pybind_stl = False
 this_project_source = f'{os.getcwd()}/include'
 this_project_include = this_project_source
 this_project_namespace_to_bind = 'testers'
 python_module_name = 'test_struct'
 
+
 def make_all_includes():
     all_includes = []
     all_include_filename = 'all_cmake_includes.hpp'
-    for filename in (glob.glob(f'{this_project_source}/**/*.hpp', recursive=True)+
-                     glob.glob(f'{this_project_source}/**/*.cpp', recursive=True)+
-                     glob.glob(f'{this_project_source}/**/*.h', recursive=True)+
-                     glob.glob(f'{this_project_source}/**/*.cc', recursive=True)+
+    for filename in (glob.glob(f'{this_project_source}/**/*.hpp', recursive=True) +
+                     glob.glob(f'{this_project_source}/**/*.cpp', recursive=True) +
+                     glob.glob(f'{this_project_source}/**/*.h', recursive=True) +
+                     glob.glob(f'{this_project_source}/**/*.cc', recursive=True) +
```

Comment:
This seems quite repetitive and repetitive... So how about something like this:
```Python
for ext in 'hpp cpp h cc c'.split():
   for filename in (glob.glob(f'{this_project_source}/**/*.' + ext, recursive=True):
```

Reply:
I agree, but I think it is better to keep it the way it is for verbosity in this example.